stable/prometheus-adapter/ci/resource-rules-values.yaml

```diff
@@ -0,0 +1,27 @@
+rules:
```

Comment:
You might as well remove this file. GKE already deploys metrics-server so we can't deploy our apiservice to test. This is why the build is failing.